### ISSUE TITLE
Problem with WeConnect makes server stop

### ISSUE THREAD

**Issue by runebakjacobsen** [tillsteinbach/VWsFriend#49]
Thanks for the project! 

This morning I experienced an issue, looks like something to do with WeConnect. The problem is that `vwsfriend_vwsfriend_1` requires a manuel restart. (Also the issue is persisting)

```
vwsfriend_1   | CRITICAL:VWsFriend:There was a problem when communicating with WeConnect. If this problem persists please open a bug report: No Location for forwarding in response headers
vwsfriend_vwsfriend_1 exited with code 0
```

**Comment by tillsteinbach** [comment]
Yes, VW has some serious problems with the WeConnect servers right now. They are returning strange responses. I need to make the scripts more robust against such responses but it is super hard to test this as they are quite sporadic.
I will have a look in the next days.
As a workaround you could ask docker to automatically restart the container: https://docs.docker.com/compose/compose-file/compose-file-v3/#restart_policy

**Comment by runebakjacobsen** [comment]
Thanks so much, will look into the docker workaround 👍  Hope VW gets their system in order 😆
